pkg/roachprod/cloud/cluster_cloud.go

```diff
@@ -360,6 +361,8 @@ func GrowCluster(l *logger.Logger, c *Cluster, NumNodes int) error {
 
 // DestroyCluster TODO(peter): document
 func DestroyCluster(l *logger.Logger, c *Cluster) error {
+	_ = promhelperclient.NewPromClient().DeleteClusterConfig(context.Background(),
```

Comment:
Could we run into an issue here if this cluster is destroyed but not on the default project, since a non-default project would not have created an entry in the first place?

Reply:
few more refactoring and this could be done. Thanks!